kitsune/wiki/views.py

```diff
@@ -125,6 +125,30 @@ def document(request, document_slug, template=None, document=None):
             # and OK to fall back to parent (parent is approved).
             fallback_reason = 'no_translation'
 
+    # If any local has fallback locals and the docmunet is missing localization in that local
+    # So show the fallback local's document. There can be any number of fallback locals.
+    # Fallback locals can be add in kitsune/wiki/config.py
+    if fallback_reason == 'no_translation' and request.LANGUAGE_CODE in FALLBACK_LOCALS:
+        # Get fallback locals for the requested local
+        fallback_local_code = FALLBACK_LOCALS[request.LANGUAGE_CODE]
+        # Get the locals in which the document is already translated and must have current revision
+        translated_locales = (doc.translations.all()
+                                              .exclude(current_revision=None)
+                                              .values_list('locale', flat=True))
+        # Check translation of fallback locals according to their order
+        # If the document is translated into any fallback local, show in that local
+        # Will regard the ordering sequence to choose which local should be show
+        for fallback_local in fallback_local_code:
+            if fallback_local in translated_locales:
+                translation = doc.translated_to(fallback_local)
+                doc = Document.objects.get(locale=fallback_local,
+                                           slug=translation.slug)
+                fallback_reason = 'fallback_local'   # For showing note to users
+                break
+            else:
+                doc = get_object_or_404(Document, locale=settings.WIKI_DEFAULT_LANGUAGE,
```

Comment:
This `else` needs to be de-dented by one left so that it is associated with the `for` on line 141, not the `if` on line 142. The way you have it here, only the first fallback locale will be considered.